config/locales/account/fr.yml

```
@@ -114,4 +114,6 @@ fr:
     security:
       link: En apprendre davantage dans le Centre d'aide
       text: L'information de votre profil est verrouillée pour votre sécurité.
+    terms: J'accepte les conditions d'utilisation de login.gov et les pratiques de
```

FYI if #4954 lands before this, the plain `'` apostrophes would become a CI build failure if not corrected.

Could fix this now or at least will need a merge / rebase when #4954 is merged.

```suggestion
    terms: J’accepte les conditions d’utilisation de login.gov et les pratiques de
```